src/include/cmark-gfm_config.h

```diff
@@ -15,6 +15,15 @@ extern "C" {
   typedef char bool;
 #endif
 
+#if defined(__has_include)
+#  if __has_include(<unistd.h>)
+#    define HAVE_UNISTD_H
+#  endif
+#elif defined(unix) || defined(__unix__) || defined(__unix) || defined(__APPLE__)
+  // Apple Clang does not define any of the unix symbols, even though it provides unistd.h
+#  define HAVE_UNISTD_H
+#endif
```

Comment:
Any reason to not just prefer this with or without CMake?

Reply:
I feel like this can be somewhat less accurate, though it could be used instead of the header check, both in CMake and in the prebuilt header. Do you know if MinGW or Cygwin define one of the `__unix__` flags? They should provide `<unistd.h>`, if i remember right.

Reply:
Cygwin does define `unix`, `__unix` and `__unix__`.  MinGW does not. That seems reasonable though.

Reply:
Should i leave the `__has_include` check here for the Swift package, then? I can leave the header check in the CMake script for users building this fork standalone.

Reply:
Yeah, I think that leaving this check as is should be good.  I can't resolve the conversation (likely due to missing permissions on the repo).